src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -131,6 +131,67 @@ def provisionInfrastructure(target, item, parameters) {
   }
 }
 
+def generatePrNames(dbName, prCode) {
+  def prSchema = "pr$prCode"
+  def prUser = "${dbName}_$prSchema"
+  return [prSchema, prUser]
+}
+
+def runPsqlCommand(dbHost, dbUser, dbName, sqlCmd) {
+  sh returnStdout: true, script: "psql --host=$dbHost --username=$dbUser --dbname=$dbName --no-password --command=\"$sqlCmd;\""
+}
```

Comment:
For this, and the previous function, I think they're only called internally for which I instituted the double underscore prefix notation. This isn't documented (probably should be), but indicates which methods shouldn't be called externally and is probably a convention we should stick to. Happy to discuss this though